auslib/admin/views/scheduled_changes.py

```diff
@@ -0,0 +1,214 @@
+                else:
+                    r[k.replace("base_", "")] = v
+            ret["scheduled_changes"].append(r)
+        return jsonify(ret)
+
+    def _post(self, form, transaction, changed_by):
+        if not form.validate():
+            self.log.warning("Bad input: %s", form.errors)
+            return Response(status=400, response=json.dumps(form.errors))
+
+        try:
+            sc_id = self.sc_table.insert(changed_by, transaction, **form.data)
+        except ValueError as e:
+            self.log.warning("Bad input: %s", e)
+            return Response(status=400, response=json.dumps({"exception": e.args}))
+
+        return jsonify({"sc_id": sc_id})
+
+
+class ScheduledChangeView(AdminView):
+    """/scheduled_changes/:namespace/:sc_id"""
+
+    def __init__(self, namespace, table):
+        self.namespace = namespace
+        self.path = "/scheduled_changes/%s/:sc_id" % namespace
+        self.table = table
+        self.sc_table = table.scheduled_changes
+        super(ScheduledChangeView, self).__init__()
+
+    def _post(self, sc_id, form, transaction, changed_by):
+        if not form.validate():
+            self.log.warning("Bad input: %s", form.errors)
+            return Response(status=400, response=json.dumps(form.errors))
+
+        what = dict()
+        # We need to be able to support changing AND removing things
+        # and because of how Flask's request object and WTForm's defaults work
+        # this gets a little hairy.
+        for k, v in form.data.iteritems():
+            # sc_data_version is a "special" column, in that it's not part of the
+            # primary data, and shouldn't be updatable by the user.
+            if k == "sc_data_version":
+                continue
+            # If the key is not present in the request we treat it as a no-op
+            # and shouldn't modify the data for that key.
+            # If the key is present we should modify the data as requested.
+            # If a value is an empty string, we should remove that restriction
+            # from the rule (aka, set as NULL in the db). The underlying Form
+            # will have already converted it to None, so we can treat it the
+            # same as a modification here.
+            if (request.json and k in request.json):
+                what[k] = v
+
+        try:
+            self.sc_table.update({"sc_id": sc_id}, what, changed_by, form.sc_data_version.data, transaction)
+        except ValueError as e:
+            self.log.warning("Bad input: %s", e)
+            return Response(status=400, response=json.dumps({"exception": e.args}))
+
+        sc = self.sc_table.select(where={"sc_id": sc_id}, transaction=transaction, columns=["data_version"])[0]
+        new_data_version = sc['data_version']
+        response = make_response(json.dumps(dict(new_data_version=new_data_version)))
+        response.headers['Content-Type'] = 'application/json'
+        return response
```

Comment:
There's some cases (eg: https://github.com/mozilla/balrog/blob/cf68278bf46eddbfff895697ce0081a908d8ee9e/auslib/admin/views/releases.py#L491) where we need to set a different status code, and jsonify doesn't allow us to do that. In this case I don't think there's a reason not to just use jsonify...